nodejs/eks/nodegroup.ts

```diff
@@ -942,9 +977,13 @@ function createNodeGroupInternal(
             instanceType: args.instanceType || "t3.medium",
             iamInstanceProfile: instanceProfile,
             keyName: keyName,
+            // This apply is necessary in s.t. the launchConfiguration picks up a
+            // dependency on the eksClusterIngressRule. The nodes may fail to
+            // connect to the cluster if we attempt to create them before the
+            // ingress rule is applied.
             securityGroups: pulumi
-                .all([nodeSecurityGroupId, extraNodeSecurityGroupIds])
-                .apply(([sg, extraSG]) => [sg, ...extraSG]),
+                .all([nodeSecurityGroupId, extraNodeSecurityGroupIds, eksClusterIngressRuleId])
+                .apply(([sg, extraSG, ingress]) => [sg, ...extraSG]),
```

Comment:
ingress is ignored here, just added as a dependency, ah the comment explains it. Wonder if TypeScript lets mark unused variables as _ or _ingress.

Reply:
We can mark it as `_` 👍